.ci/scripts/test.bat

```diff
@@ -1,14 +1,12 @@
-set
-set GO111MODULE=off
-go get -u github.com/elastic/go-licenser
-set GO111MODULE=on
+
 go mod verify
-go-licenser -d
-go run .ci/scripts/check_format.go
-go run .ci/scripts/check_lint.go
-
-mkdir -p build
-SET OUT_FILE=build\output-report.out
-go test "./..." -v > %OUT_FILE% | type %OUT_FILE%
-go get -v -u github.com/jstemmer/go-junit-report
-go-junit-report > build\junit-%GO_VERSION%.xml < %OUT_FILE%
+go run github.com/elastic/go-licenser@latest -d
```

Comment:
I think these commands need some error handling to propagate the status code to the script's exit code. Otherwise the CI status will be green despite a failure in this script. Lots of examples in http://steve-jansen.github.io/guides/windows-batch-scripting/part-3-return-codes.html.

```suggestion
go run github.com/elastic/go-licenser@latest -d || EXIT /B 1
```

Reply:
Thank you for spotting this! :bow: